com.github.danrabbit.harvey.yml

```diff
@@ -0,0 +1,26 @@
+id: com.github.danrabbit.harvey
+base: io.elementary.Loki.BaseApp
+base-version: stable
+runtime: org.freedesktop.Platform
+runtime-version: '1.6'
```

Comment:
18.08

Reply:
The Loki base-app would need to get updated too in that case…